### PR TITLE
#786 - Revise tests of polyhedra

### LINKED ISSUE #786: Replace empty polytopes in unit_Polytope
Some of the polytopes are empty, which invalidates all the checks.

### PR DESCRIPTION
**Pull request by schillic**
Closes #786.

### CODE REVIEW

File: test/unit_Polyhedron.jl [hunk 1]
```diff
@@ -112,27 +112,26 @@ if test_suite_polyhedra
...
 
         # intersection with polytope
         A = [N(1) N(0); N(0) N(1); N(-1) N(-1)]
         b = N[1, 1, -1]
         p = HPolyhedron(A, b)
         b = BallInf(N[1, 1], N(0.5))
-        cap = intersection(p, b)
-        cap = intersection(b, p)
+        intersection(p, b)
+        intersection(b, p)
```
Comment (mforets) [inline test/unit_Polyhedron.jl]: the intention was to test something here or just to check that `intersection` can dispatch on `HPolyhedron` and `BallInf`?

Comment (schillic) [reply test/unit_Polyhedron.jl]: I do not know. I just removed the return value because we never used it :wink:

Comment (mforets) [reply test/unit_Polyhedron.jl]: 😆  we could check that the obtained sets are the same, for instance

Comment (schillic) [reply test/unit_Polyhedron.jl]: Okay, see 1531.

**Pull request merged**